app.js

```diff
@@ -3,6 +3,20 @@ const closeMenu = document.querySelector('#close-img');
 const mobileMenu = document.querySelector('#mobile-menu');
 const menuLinks = document.querySelectorAll('#mobile-menu-list li');
 const sectionBlur = document.querySelectorAll('footer, header, main');
+const closeWorks = document.querySelector('#close-works');
+const works = document.querySelectorAll('#works>div');
+const lightboxWorks = document.querySelector('#works-lightbox');
+const lightboxContent = {
+  title: document.querySelector('#works-lightbox-tittle'),
+  client: document.querySelector('#works-lightbox-client'),
+  role: document.querySelector('#works-lightbox-role'),
+  year: document.querySelector('#works-lightbox-year'),
+  image: document.querySelector('#works-lightbox-image'),
+  body: document.querySelector('#works-lightbox-body'),
+  skills: document.querySelector('#works-lightbox-skills'),
+};
+
```

Comment:
- [ ] Please according to the project requirement, kindly ensure to use JavaScript code and implements an array of objects to store JavaScript objects containing the information for each project. ℹ️ 

## Project Requirement
![q3](https://user-images.githubusercontent.com/103386183/209191661-2280f44b-f4a7-4438-b562-19781ed36c3b.png)